packages/widget/src/widget/Widget.tsx

```diff
@@ -189,6 +189,6 @@ const WidgetContainer = styled.div`
   position: relative;
   padding: 0 10px;
   @media (max-width: 500px) {
-    width: 100%;
+    width: 100vw;
```

Comment:
this works better and doesn't require the parent container to be 100% on mobile screen sizes for the mobile page widths to be consistent for all pages